### ISSUE TITLE
swagger-ui not included in published crate

### ISSUE THREAD

**Issue by dbanty** [paperclip-rs/paperclip#353]
When I try to use the `swagger-ui` feature from crates.io, the build fails as it cannot include the submodule directory. I assume that the submodule was not checked out whenever the build happened ([see this issue about warning on that](https://github.com/rust-lang/cargo/issues/8635)). The feature works fine when I include paperclip as a git dependency instead.

### This Works

```toml
paperclip = { version = "0.6.1", features = ["actix", "v3", "swagger-ui"], git = "https://github.com/paperclip-rs/paperclip" }
```

### This Fails

```toml
paperclip = { version = "0.6.1", features = ["actix", "v3", "swagger-ui"] }
```

with this message

```
% cargo build           
   Compiling paperclip-actix v0.4.1
error: proc-macro derive panicked
  --> /Users/dylan/.cargo/registry/src/github.com-1ecc6299db9ec823/paperclip-actix-0.4.1/src/lib.rs:48:28
   |
48 | static SWAGGER_DIST: Dir = include_dir!("../swagger-ui/dist");
   |                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: message: "/Users/dylan/.cargo/registry/src/github.com-1ecc6299db9ec823/paperclip-actix-0.4.1/../swagger-ui/dist" doesn't exist
   = note: this error originates in the macro `include_dir` (in Nightly builds, run with -Z macro-backtrace for more info)

error: cannot find macro `proc_macro_call` in this scope
  --> /Users/dylan/.cargo/registry/src/github.com-1ecc6299db9ec823/paperclip-actix-0.4.1/src/lib.rs:48:28
   |
48 | static SWAGGER_DIST: Dir = include_dir!("../swagger-ui/dist");
   |                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: this error originates in the macro `include_dir` (in Nightly builds, run with -Z macro-backtrace for more info)

error: could not compile `paperclip-actix` due to 2 previous errors

```

**Comment by tiagolobocastro** [comment]
I wonder if the reason for this is that the submodule is not actually within the paperclip-actix plugin. I wonder if adding a symlink to the paperclip-actix directory would work?
I don't want to keep pushing new versions to test it though so I tried running a local registry but when I try to publish to it fails :/

**Comment by tiagolobocastro** [comment]
Should be fixed by paperclip-actix plugin 0.4.2
